ote/src/cljs/ote/views/transport_service_common.cljs

```diff
@@ -77,7 +78,16 @@
                    {:name ::t-service/license :type :string :width "21%"}
                    {:name ::t-service/license-url :type :string :width "21%"}]
     :delete? true
-    :add-label (tr [:buttons :add-external-interface])}))
+    :add-label (tr [:buttons :add-external-interface])}
+
+   {:name ::t-service/notice-external-interfaces?
+    :type :checkbox
+    :required? true
+    :style style-form/padding-top
+    :validate [[:checked?]]
+    }
```

Comment:
format ending parens properly

Reply:
why name it `::t-service/notice-external-interfaces?` this field is just for the frontend and not in the service db model... this should not be sent to the server.